product_docs/docs/jdbc_connector/42.3.3.1/05_using_the_advanced_server_jdbc_connector_with_java_applications/02_connecting_to_the_database/02_preferring_synchronous_secondary_database_servers.mdx

```diff
@@ -76,23 +76,23 @@ In the example that follows, the:
 
 In the primary database server’s `pg_hba.conf` file, there must be a replication entry for each unique replication database `USER/ADDRESS` combination for all secondary database servers. In the following example, the database superuser `enterprisedb` is used as the replication database user for both the `secondary1` database server on `192.168.2.22` and the `secondary2` database server that is local relative to the primary.
 
-```text
+```
```

Comment:
Let's use "text" instead of no language here and in any other blocks where you may have removed "text" as that indicates we made a conscious choice.